x-pack/plugins/observability/server/lib/rules/threshold/register_threshold_rule_type.ts

```diff
@@ -183,11 +182,6 @@ export function thresholdRuleType(
         { name: 'value', description: valueActionVariableDescription },
         { name: 'metric', description: metricActionVariableDescription },
         { name: 'threshold', description: thresholdActionVariableDescription },
-        {
-          name: 'viewInAppUrl',
```

Comment:
Are we not setting the viewInAppUrl on the alert context anywhere from the executor?

Reply:
@kdelemme, We are pulling it off from the action variable because we don't have a URL for it, yet. Will will have a different URL based on the params e.g. "Doc count" The viewInApp should lead to the Logs.